docs/source/configuration_file.rst

```diff
@@ -324,11 +324,12 @@ to the tailored architecture.
 
 - ``name``: ``Unet`` (default)
 - ``dropout_rate``: Float (e.g. 0.4).
-- ``batch_norm_momentum``: Float (e.g. 0.1).
+- ``bn_momentum``: Float (e.g. 0.1).
```

Comment:
could you also add a description about this field? (and maybe a ref)

Reply:
Added here: 77150bb . I also changed the default value in our config files to match the one defined by PyTorch (9fc873f)